modules/editor/message/PrimaryContentEditor.tsx

```diff
@@ -25,28 +32,43 @@ export function PrimaryContentEditor(props: PrimaryContentEditorProps) {
         error={form.field("content").error}
         {...form.field("content").inputProps}
       />
-      <RowContainer>
-        <InputField
-          id={`_${message.id}_username`}
-          label="Username"
-          maxLength={80}
-          error={form.field("username").error}
-          {...form.field("username").inputProps}
-        />
-        <InputField
-          id={`_${message.id}_avatar`}
-          label="Avatar URL"
-          error={form.field("avatar").error}
-          {...form.field("avatar").inputProps}
-        />
-      </RowContainer>
-      <FileInputField
-        id={`_${message.id}_files`}
-        label="Files"
-        maxSize={8 * 1024 ** 2}
-        value={message.files}
-        onChange={files => message.set("files", files)}
-      />
+      <Section variant="large" color={black} hasError={false} name="Extras">
```

Comment:
Please omit the color

Reply:
Without color left corner of section looks weird
![image](https://user-images.githubusercontent.com/16510197/108513732-0c4df180-72d4-11eb-9605-d5cb52dfec1c.png)

Reply:
Use a small section instead, a large one would look too similar to an embed